Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -254,6 +254,32 @@ private void HandleSmartEquip(ICommonSession? session, string equipmentSlot)
             if (!_inventorySystem.TryGetSlotEntity(plyEnt, equipmentSlot, out var slotEntity) ||
                 !TryComp(slotEntity, out ServerStorageComponent? storageComponent))
             {
+                if (_inventorySystem.TryGetSlot(plyEnt, equipmentSlot, out var slotDefinition) &&
+                    TryComp(plyEnt, out InventoryComponent? inv))
```

Comment:
`HasSlot` and `HasComp`, respectively to lines 257/258

Reply:
Realized i don't even need to check if player has an inventory (the original `HandleSmartEquip` never does it), so i removed it in the next commit.